<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArrowsLengthSettingEditWidget</class>
 <widget class="QWidget" name="ArrowsLengthSettingEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="lengthGroupBox">
     <property name="title">
      <string>Length</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="lengthAutoCheckBox">
        <property name="text">
         <string>Auto</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="stdValueLabel">
            <property name="text">
             <string>Standard value:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="stdValueSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="decimals">
             <number>6</number>
            </property>
            <property name="minimum">
             <double>0.001000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="stdValueUnitLabel">
            <property name="text">
             <string>[m/s]</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="legendLenLabel">
            <property name="text">
             <string>Length on screen:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="legendLengthSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="legendLenUnitLabel">
            <property name="text">
             <string>[pixel]</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="minValueLabel">
            <property name="text">
             <string>Minimum value to draw:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="minValueSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="decimals">
             <number>8</number>
            </property>
            <property name="minimum">
             <double>0.001000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="minValueUnitLabel">
            <property name="text">
             <string>[m/s]</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lengthAutoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>stdValueSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lengthAutoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>legendLengthSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>36</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lengthAutoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>minValueSpinBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
